weave

```diff
@@ -2251,7 +2248,6 @@ EOF
         ;;
     stop)
         [ $# -eq 0 ] || usage
-        plugin_disabled || stop_plugin
```

Comment:
I was thinking about giving explicit instructions in release notes instead. But anyway, I don't have a strong opinion about it though.